src/pages/nft/class/_classId/index.vue

```diff
@@ -418,7 +419,10 @@ export default {
       this.updateNFTHistory();
       this.lazyFetchLIKEPrice();
       this.fetchUserCollectedCount();
-      const blockingPromises = [this.fetchISCNMetadata()];
+      const blockingPromises = [
+        this.fetchISCNMetadata(),
+        this.fetchNFTBookPriceByClassId(this.classId),
```

Comment:
So we try to fetch a `book price` for all NFT? Can we safely assume we only have to fetch for those under a `NFT book` collection?